lnrpc/routerrpc/router.proto

```diff
@@ -105,6 +105,13 @@ message SendPaymentRequest {
 
     /// If set, circular payments to self are permitted.
     bool allow_self_payment = 15;
+
+    /**
+    Features assumed to be supported by the final node. All transitive feature
+    depdencies must also be set properly. For a given feature bit pair, either
+    optional or remote may be set, but not both.
```

Comment:
Is it possible to define an enum in the proto for the feature bits, to prevent mistakes?

Reply:
done. it should also be possible for people to send custom features by casting using `lnrpc.FeatureBit(x)`